ckanext/s3filestore/plugin.py

```diff
@@ -68,6 +70,11 @@ def before_map(self, map):
                 m.connect('resource_download',
                           '/dataset/{id}/resource/{resource_id}/download/{filename}',
                           action='resource_download')
+            #Allow fallback to access old files
+            if not self.use_filename:
+                m.connect('resource_download',
+                          '/dataset/{id}/resource/{resource_id}/orig_download/{filename}',
```

Comment:
What actually calls the `orig_download` URL?

Reply:
externals who know about it